torch/csrc/lazy/ts_backend/ir_builder.h

```diff
@@ -0,0 +1,68 @@
+#pragma once
+
+#include <torch/csrc/lazy/core/ir.h>
+#include <torch/csrc/lazy/core/ir_builder.h>
+#include <torch/csrc/lazy/core/internal_ops/ltc_ops.h>
+#include <torch/csrc/lazy/core/shape_inference.h>
+#include <torch/csrc/lazy/ts_backend/ts_node.h>
+#include <torch/csrc/lazy/ts_backend/dynamic_ir.h>
+#include <torch/csrc/lazy/ts_backend/view_ops/narrow.h>
+#include <torch/csrc/lazy/ts_backend/view_ops/select_view_update.h>
+#include <torch/csrc/lazy/ts_backend/view_ops/as_strided_view_update.h>
+#include <torch/csrc/lazy/ts_backend/view_ops/permute.h>
+#include <torch/csrc/lazy/ts_backend/view_ops/diagonal_view_update.h>
+#include <torch/csrc/lazy/ts_backend/view_ops/resize.h>
+#include <torch/csrc/lazy/ts_backend/view_ops/squeeze.h>
+#include <torch/csrc/lazy/ts_backend/view_ops/diagonal.h>
+#include <torch/csrc/lazy/ts_backend/view_ops/narrow_view_update.h>
+#include <torch/csrc/lazy/ts_backend/view_ops/as_strided.h>
+#include <torch/csrc/lazy/ts_backend/view_ops/unsqueeze.h>
+#include <torch/csrc/lazy/ts_backend/view_ops/select.h>
+#include <torch/csrc/lazy/ts_backend/view_ops/view.h>
+#include <torch/csrc/lazy/ts_backend/ops/cast.h>
+#include <torch/csrc/lazy/ts_backend/ops/device_data.h>
+#include <torch/csrc/lazy/ts_backend/ops/generic.h>
+#include <torch/csrc/lazy/ts_backend/ops/batch_norm_ops.h>
```

Comment:
out of curiosity, is this header really needed?

Reply:
good catch. Its not strictly needed here. It was only included as this file was originally autogen'd and it just included all of the ops headers. It shouldn't hurt to include it though. I don't think its worth restarting CI over. It can probably be cleaned up at some future time if need be